executor.py

```diff
@@ -59,19 +58,18 @@ def __init__(
         self.logger = JinaLogger(
             getattr(self.metas, "name", self.__class__.__name__)
         ).logger
-        self.max_num_training_points = max_num_training_points
         self.cluster_center = cluster_center
         self.iter = iter_steps
         self.metric = 'euclidean'
-        self.subspaces = num_subspaces
-        if num_codeword and num_codeword > 256:
+        self.subspaces = subspaces
+        if codewords and codewords > 256:
             self.logger.warning(
```

Comment:
So this is a hard limit for this parameter?

Reply:
Yes, its from Rii library